src/Evolution/Systems/Cce/Equations.cpp

```diff
@@ -24,6 +24,74 @@
 // IWYU pragma: no_forward_declare SpinWeighted
 
 namespace Cce {
+
+namespace detail {
+void klein_gordon_rhs_npsi1(
+    const gsl::not_null<SpinWeighted<ComplexDataVector, 0>*> result,
+    const SpinWeighted<ComplexDataVector, 1>& eth_beta,
+    const SpinWeighted<ComplexDataVector, 1>& eth_kg_psi,
+    const SpinWeighted<ComplexDataVector, 0>& eth_ethbar_kg_psi,
+    const SpinWeighted<ComplexDataVector, 0>& bondi_k) {
+  (*result).data() = (2. * real(eth_beta.data() * conj(eth_kg_psi).data()) +
+                      eth_ethbar_kg_psi.data()) *
```

Comment:
Do eth and ethbar commute? In the docs you have it written as $\bar\eth\eth$

Reply:
They commute while acting on spin-weighted 0 quantities, say the scalar field. I've updated the doc